src/App.js

```diff
@@ -179,70 +164,28 @@ const App = () => {
             <h1 className="display-1 text-dark">Matt's budget</h1>
           </div>
           <div className="col-md-3">
-            <MonthSelector
-              expenses={expenses}
-              selectedMonth={selectedMonth}
-              setSelectedMonth={setSelectedMonth}
-            />
+            <MonthSelector expenses={expenses} selectedMonth={selectedMonth} setSelectedMonth={setSelectedMonth} />
```

Comment:
I think this is another example of where the wrapping actually helps, in my opinion.